ISM43362/ISM43362.cpp

```diff
@@ -196,6 +196,61 @@ bool ISM43362::dhcp(bool enabled)
     return (_parser.send("C4=%d", enabled ? 1 : 0) && check_response());
 }
 
+/*
+  * @brief  Update modem_country_code.
+  * @param  country_codeP: User's country code.
+  * @retval boolean value : false ->error
+
+WiFi modem is only able to receive 4 country codes: CA, US, FR and JP.
+CA, US -> 11 channels
+FR     -> 13 channels
+JP     -> 14 channels
+
+For each input country code(country_codeP), function is specifying
+the number of channels : 11, 13 or 14.
+Function is updating private variable modem_country_code.
+
+For instance, country_codeP=VI, function updates modem_country_code to US.
```

Comment:
I don't understand this comment what is "VI" ?

Reply:
Comment has been modified.